sky/packages/sky/lib/sky_tool

```diff
@@ -447,6 +488,27 @@ class AndroidDevice(object):
         logging.info(' '.join(cmd))
         subprocess.check_output(cmd)
 
+    def logs(self):
+        def do_logs():
+            cmd = [
+                self.adb_path,
+                'logcat',
+                '-s',
+                'sky',
+                'chromium',
```

Comment:
The two invocations seem to give identical output, correctly including chromium lines.

It looks like -s is for silencing the default filter, and then everything after that is interpreted as the set of filters, which explains why both syntaxes work.